SQL/tgmc-schema.sql

```diff
@@ -100,7 +100,7 @@ CREATE TABLE IF NOT EXISTS `death` (
   `server_ip` int(10) unsigned NOT NULL,
   `server_port` smallint(5) unsigned NOT NULL,
   `round_id` int(11) NOT NULL,
-  `tod` datetime NOT NULL COMMENT 'Time of death',
+  `timeofdeath` datetime NOT NULL COMMENT 'Time of death',
```

Comment:
Revert this. This is just the name of the table in the database, not affected by any in-game variables.